### PR TITLE
feat: create webpage that allows user input of files and data

### PR DESCRIPTION
**Pull request by siric-4321**
# Pull request

**Resolves:** #42

### CODE REVIEW

File: new
Comment (tclasen) [inline new:1]: Please delete this file.

**Review by tclasen — CHANGES_REQUESTED** [review]

File: new
Comment (siric-4321) [reply new:1]: Which file particularly?

Comment (tclasen) [reply new:1]: The comment was left on the file in question, and is at the top of this chat dialog box. You can click on the name "new" above my comment to go to the file of the same name. In this PR you can also navigate to "Files Changed" to see that you created a file with no contents and no purpose in existing.

File: darkmoon/index.html
Comment (tclasen) [inline darkmoon/index.html:1]: Please relocate the html and css files into the templates folder added by the jinja PR that is open.

File: darkmoon/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    </h1>
+    <h1>
+
+        <label for="username" class="bu" data-icon="u"> </label>
+
+        <input id="bu" name="username" required="required" type="text" placeholder="Build" />
+    </h1>
+
+    <form action="/action_page.php">
```
Comment (tclasen) [inline darkmoon/index.html:45]: Please remove references to the PHP targets and point these actions to their respective endpoints within the Darkmoon API.

File: darkmoon/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```
Comment (tclasen) [inline darkmoon/index.html:4]: It looks like a bunch of stuff got duplicated on copy / paste into the template?

File: darkmoon/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+
+    </div>
+
+    <h1>
+
+        <label for="username" class="opSystem" data-icon="u"></label>
```
Comment (tclasen) [inline darkmoon/index.html:25]: What are these fields being used for?

**Review by tclasen — CHANGES_REQUESTED** [review]

File: new
Comment (siric-4321) [reply new:1]: Just deleted it and committed it.

Comment (tclasen) [reply new:1]: Don't forget to mark this conversation thread as "resolved" once you have completed the task.

File: darkmoon/index.html (hunk 3 above)
Comment (siric-4321) [reply darkmoon/index.html:25]: It got duplicated by they link the seperate CSS file to the html. Without them there wouldn't be the CSS.

File: darkmoon/index.html (hunk 2 above)
Comment (siric-4321) [reply darkmoon/index.html:4]: Cleared out the duplicates!

**Comment by DoodlePooge** [comment]
This PR is blocked by #45 
The webpage itself, from what I saw, looked decent.

**Comment by pelavarthi** [comment]
looks okay for owner review

**Comment by DoodlePooge** [comment]
I see that this PR is linked for 2 tickets, making the webpage, and adding the CSS framework. No framework is being used here, which is fine, but if we wanted to utilize a framework it would be a lot easier to do it now before we hardcode a lot of CSS...

**Review by DoodlePooge — CHANGES_REQUESTED** [review]

**Pull request closed without merging**